NFT-Marketplace-Part-2.md

```diff
@@ -147,89 +149,91 @@ There are three main files we will be touching in this project, and should take
       apiVersion: 0.0.5
       language: wasm/assemblyscript
       entities:
-        - ListingCanceled
+        - ListingCancelled
         - ListingCreated
         - ListingPurchased
         - ListingUpdated
       abis:
         - name: NFTMarketplace
           file: ./abis/NFTMarketplace.json
       eventHandlers:
-        - event: ListingCanceled(address,uint256,address)
-          handler: handleListingCanceled
+        - event: ListingCancelled(address,uint256,address)
+          handler: handleListingCancelled
         - event: ListingCreated(address,uint256,uint256,address)
           handler: handleListingCreated
         - event: ListingPurchased(address,uint256,address,address)
           handler: handleListingPurchased
         - event: ListingUpdated(address,uint256,uint256,address)
           handler: handleListingUpdated
-      file: ./src/mapping.ts
+      file: ./src/nft-marketplace.ts
 ```
 
 Few things to note here:
 
 - `dataSources` is the main block we want to focus on, and is literally used to define the data source for our indexer
 - `source` block defines the contract address and a reference to its ABI. Note that the ABI is only given a name, as the actual file path is referenced to later in the `abis` block
 - `eventHandlers` has the definitions of all the events our contract had, which the CLI was able to automatically generate from looking at the ABI. Each event also has a `handler` defined, which will be the function name within our script to handle data coming from that event.
-- `file` references `src/mapping.ts` - this is where we will write our script.
+- `file` references `/src/nft-marketplace.ts` - this is where we will write our script.
 
 For now, we will just make one small change to this manifest. Since The Graph works by scanning every block of the blockchain trying to find events which match your data sources, by default, it will start scanning from the genesis block of the blockchain.
 
+*`A Genesis Block` is the name given to the first block of a cryptocurrency. In this case, it's the block 0 of Celo.*
+
 However, that will take a lot of time, as there are millions of blocks which we know for sure don't have any events we are interested in, since our contract was only recently deployed.
 
 Therefore, to speed it up, we can define a `startBlock` field to tell The Graph where to start scanning from. We will set this value to be the block in which our contract was deployed.
 
 Search up your contract address on [CeloScan](https://alfajores.celoscan.io), find the `Contract Creation` transaction, copy the block number of that transaction, and add the following line to your manifest:
 
 ```diff
-specVersion: 0.0.2
+specVersion: 0.0.4
 schema:
   file: ./schema.graphql
 dataSources:
   - kind: ethereum
     name: NFTMarketplace
     network: celo-alfajores
     source:
-      address: "0x9014DD98Cd14B26c76069356247cE6d762018220"
+      address: "0x627f152f97d431B844604B4421313CA979712006"
```

Comment:
Can you also make this change (and other code changes) in `celo-nft-marketplace/subgraph` of this repo? Thanks a lot!

Reply:
Ideally do this as part of this same PR so easier to review